internal/store/store.go

```diff
@@ -32,11 +33,14 @@ func New(dsn string, logger logrus.FieldLogger) (*SQLStore, error) {
 
 	switch strings.ToLower(url.Scheme) {
 	case "sqlite", "sqlite3":
```

Comment:
Why bother supporting sqlite?

Reply:
Primarily for local development -- it's slightly easier to manage/switch between a local sqlite than a running Postgres instance.